src/Features/Core/Portable/CodeFixes/Suppression/WrapperCodeFixProvider.cs

```diff
@@ -12,10 +12,10 @@ internal sealed class WrapperCodeFixProvider : CodeFixProvider
         private readonly ImmutableArray<string> _originalDiagnosticIds;
         private readonly ISuppressionFixProvider _suppressionFixProvider;
 
-        public WrapperCodeFixProvider(ISuppressionFixProvider suppressionFixProvider, ImmutableArray<Diagnostic> originalDiagnostics)
+        public WrapperCodeFixProvider(ISuppressionFixProvider suppressionFixProvider, ImmutableArray<string> diagnosticIds)
```

Comment:
Can we instead just change this to take ImmutableArray<DiagnosticData> as the parameter and retain the `.Select(d => d.Id).Distinct()` invocation here?